server/tests/runs.e2e-spec.ts

```diff
@@ -338,4 +338,34 @@ describe('runs', () => {
         it('should respond with 401 if no access token is provided', () => get('runs', 401, {}, null));
 
     });
+
+    describe('GET /api/v1/runs/{runID}', () => {
+        const expects = (res: request.Response) => expect(res.body).toBeValidDto(RunDto);
+        it('should return a valid run', async () => {
+            const res = await get('runs/' + run1.id, 200);
+            expects(res);
+            expect(res.body.id).toBe(run1.id.toString()); // The id is a string for some reason, change when that's fixed
```

Comment:
This is a JSON response, so everything is a string! The `toString` call here is fine, you can just remove the comment.

Reply:
Never mind I'm an idiot. It's cus it's a bigint we serialise out to string